[2023-01-31T19:21:42.047+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: Streaming_pipeline_meli.Status_info_postgres manual__2023-01-31T16:01:06.868325+00:00 [queued]>
[2023-01-31T19:21:42.056+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: Streaming_pipeline_meli.Status_info_postgres manual__2023-01-31T16:01:06.868325+00:00 [queued]>
[2023-01-31T19:21:42.056+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-01-31T19:21:42.056+0000] {taskinstance.py:1284} INFO - Starting attempt 14 of 18
[2023-01-31T19:21:42.056+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-01-31T19:21:42.068+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): Status_info_postgres> on 2023-01-31 16:01:06.868325+00:00
[2023-01-31T19:21:42.072+0000] {standard_task_runner.py:55} INFO - Started process 17127 to run task
[2023-01-31T19:21:42.074+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Streaming_pipeline_meli', 'Status_info_postgres', 'manual__2023-01-31T16:01:06.868325+00:00', '--job-id', '1305', '--raw', '--subdir', 'DAGS_FOLDER/streamPipeline.py', '--cfg-path', '/tmp/tmplb5sr105']
[2023-01-31T19:21:42.075+0000] {standard_task_runner.py:83} INFO - Job 1305: Subtask Status_info_postgres
[2023-01-31T19:21:42.123+0000] {task_command.py:389} INFO - Running <TaskInstance: Streaming_pipeline_meli.Status_info_postgres manual__2023-01-31T16:01:06.868325+00:00 [running]> on host 15c99fd3082a
[2023-01-31T19:21:42.182+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Streaming_pipeline_meli
AIRFLOW_CTX_TASK_ID=Status_info_postgres
AIRFLOW_CTX_EXECUTION_DATE=2023-01-31T16:01:06.868325+00:00
AIRFLOW_CTX_TRY_NUMBER=14
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-31T16:01:06.868325+00:00
[2023-01-31T19:21:42.183+0000] {postgresStatus.py:11} INFO - Conexion exitosa a la base de datos: meliAnalytics
[2023-01-31T19:21:43.315+0000] {python.py:177} INFO - Done. Returned value was: <postgresStatus.DataUploader object at 0x7f12f0558c90>
[2023-01-31T19:21:43.341+0000] {xcom.py:635} ERROR - Object of type DataUploader is not JSON serializable. If you are using pickle instead of JSON for XCom, then you need to enable pickle support for XCom in your *** config or make sure to decorate your object with attr.
[2023-01-31T19:21:43.344+0000] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 72, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 2301, in xcom_push
    session=session,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 72, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/xcom.py", line 240, in set
    map_index=map_index,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/xcom.py", line 627, in serialize_value
    return json.dumps(value, cls=XComEncoder).encode("UTF-8")
  File "/usr/local/lib/python3.7/json/__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/json.py", line 176, in encode
    return super().encode(o)
  File "/usr/local/lib/python3.7/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/lib/python3.7/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/json.py", line 170, in default
    return super().default(o)
  File "/usr/local/lib/python3.7/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type DataUploader is not JSON serializable
[2023-01-31T19:21:43.366+0000] {taskinstance.py:1327} INFO - Marking task as UP_FOR_RETRY. dag_id=Streaming_pipeline_meli, task_id=Status_info_postgres, execution_date=20230131T160106, start_date=20230131T192142, end_date=20230131T192143
[2023-01-31T19:21:43.387+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 1305 for task Status_info_postgres (Object of type DataUploader is not JSON serializable; 17127)
[2023-01-31T19:21:43.410+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-01-31T19:21:43.430+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
